.gitattributes

```diff
@@ -0,0 +1,2 @@
+# These files are text and should be normalized (convert crlf =&gt; lf)
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.